video_store.py

```diff
@@ -0,0 +1,155 @@
+import requests
+import datetime
+import pprint
+
+class VideoStore:
+    def __init__(self, url, selected_video=None, selected_customer=None):
+        self.url = url
+        self.selected_video = selected_video
+        self.selected_customer = selected_customer
+    
+    def create_video(self,title="Default Video",total_inventory="Default total inventory",release_date=None):
+        query_params = {
+            "title": title,
+            "total_inventory": total_inventory,
+            "release_date": release_date
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
+    
+    def list_videos(self):
+        response = requests.get(self.url+"/videos")
+        return response.json()
+    
+    def get_video(self, title=None, id=None):
+        for video in self.list_videos():
+            if video:
+                if video["title"]==title:
+                    id = video["id"]
+                    self.selected_video = video
+                elif id == video["id"]:
+                    self.selected_video = video
+
+        if self.selected_video is None:
+            print("Could not find video by that name or id" )
+
+        '''response = requests.get(self.url+f"/videos/{id}")
+        return response.json()'''
+    
+    def list_customers(self):
+        response = requests.get(self.url+"/customers")
+        return response.json()
+
+    def get_customer_rentals(self, id):
+        response = requests.get(self.url+f"/customers/{id}/rentals")
+        print(response)
+        pprint.pprint(response.json())
```

Comment:
I suggest moving the prints out of these classes and doing the prints into the `main.py`.  This way you have 2 sections.

1.  `main.py` which handles all the user interaction.
2. `video_store_wrapper.py` where you handle all the interaction with the API.